src/webserver/js/weekly-report.js

```diff
@@ -153,7 +153,7 @@ $()
                 //loop through all status entries
                 weeklyStatus.map((status) => {
                     //populate table with statuses
-                    $('<p>ProjectId: ' + status['ProjectId'] + '</p>').appendTo("#" + status['UserId'] + " > #" + status['Day']);
+                    $('<p>Project Id: ' + status['ProjectId'] + '<br>Project name: ' + status['ProjectName'] + '</p>').appendTo("#" + status['UserId'] + " > #" + status['Day']);
```

Comment:
In der Wochenansicht soll laut Mockup soweit ich weiß nur stehen "Projekt: Projektname" und sonst nix weiter. Die ProjectId kannst du ja als data Attribut mit einhängen, dann kann man das später verwenden zum löschen von dem Status.